### DISCUSSION TITLE
[Unearth] iModel viewer displaying special characters in headers, e.g. in the Properties tab. How can we get it to display normal characters?

### DISCUSSION THREAD

**Discussion by gabe-unearth** [iTwin/itwinjs-core#2720] [Q&A] [answered]
The headers under the Properties tab are displaying special characters. Anything we should be doing differently to make sure the header text displays correctly?
![image](https://user-images.githubusercontent.com/62312126/142039485-946c2859-fc39-428f-a943-559e9ae7ac10.png)

**Comment by aruniverse** [comment]
Could you check what version of `@bentley/property-grid-react`  is getting pulled into your viewer? You could check your lock file.

**Comment by grigasp** [comment, marked as answer]
These are pseudo-localized labels, which should only be used when the `NODE_ENV` env variable is set to `development`. Unsetting the env variable should change the labels to English.

**Reply by gabe-unearth** [reply]
Thanks!
